be/src/util/thrift_rpc_helper.h

```diff
@@ -28,6 +28,10 @@ class FrontendServiceClient;
 template <class T>
 class ClientConnection;
 
+template <typename T>
```

Comment:
forward declaration may not work for this declare implementation separation. Right now it is only used in the same .cpp file, should be just fine. Just let you know.